contrib/win32/openssh/install-sshd.ps1

```diff
@@ -299,4 +307,11 @@ $accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($ssh
 $acl.SetAccessRule($accessRule)
 Set-Acl -Path $sshdlog -AclObject $acl
 
+$sshdpid = Join-Path $piddir "sshd.pid"
+if(-not (test-path $sshdpid)){ $null | Set-Content $sshdpid }
+$rights = [System.Security.AccessControl.FileSystemRights]"Read, Write"
+$accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($sshdAccount, $rights, "None", "None", "Allow")
+$acl.SetAccessRule($accessRule)
+Set-Acl -Path $sshdpid -AclObject $acl
```

Comment:
I don't think there is a need for setting ACL on the sshd.pid file..

fyi, we set the ACL on log files (sshd.log, ssh-agent.log) as they contain sensitive information.